From e5eccf7fea0e8b64472171082c05592595e1f062 Mon Sep 17 00:00:00 2001
From: Sefa Eyeoglu <contact@scrumplex.net>
Date: Tue, 20 Dec 2022 13:37:43 +0100
Subject: [PATCH] Add install section to meson build

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>
---
 meson.build | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/meson.build b/meson.build
index 58182ca..06d7117 100644
--- a/meson.build
+++ b/meson.build
@@ -9,3 +9,14 @@ vkroots_dep = declare_dependency(
 )
 
 meson.override_dependency('vkroots', vkroots_dep)
+
+install_headers('vkroots.h')
+
+pkgconfig = import('pkgconfig')
+pkgconfig.generate(
+  version: meson.project_version(),
+  filebase: meson.project_name(),
+  name: meson.project_name(),
+  description: 'Vulkan layers library',
+  subdirs: '.'
+)
